File: Brute Force/linearsearch.c
#include <stdio.h>

int linsearch(int arr[], int n, int k){
    int newarr[n+1];
    for(int i = 0; i < n; i++){
        newarr[i] = arr[i];
    }
    newarr[n] = k;
    int i = 0;
    while(i < n && newarr[i] != k){
        i++;
    }
    if(i < n){
        return i;
    }
    return -1;
}

int main(){
    printf("Enter the size of the array: ");
    int n, k;
    scanf("%d", &n);
    int arr[n];
    printf("Enter the elements of the array: ");
    for(int i = 0; i < n; i++){
        scanf("%d", &arr[i]);
    }
    printf("Enter the element to be found: ");
    scanf("%d", &k);
    int x = linsearch(arr, n, k);
    if(x == -1){
        printf("Element not found");
    }
    else{
        printf("Element found at: %d", x);
    }
    return 0;
}

File: Brute Force/linearsearch.py
# selection sort
# code by Akshat Aryan

import numpy as np

def linsearch(arr, k):
    np.append(arr, k)
    i = 0
    while i < len(arr) and arr[i] != k:
        i += 1
    if i < len(arr):
        return i
    return -1

if __name__ == "__main__":
    arr = np.array(list(map(int, input("Enter the elements of the array:").split())))
    k = int(input("Enter the element to be searched for:"))
    x = linsearch(arr, k)
    if x == -1:
        print("Element not found.")
    else:
        print(f"Element found at {x}") 